### ISSUE TITLE
Wrong upload stats

### ISSUE THREAD

**Issue by vov4ik08** [kartik-v/bootstrap-fileinput#1663]
## Prerequisites
- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest `master` branch of bootstrap-fileinput.
- [x] This is not an usage question. I confirm having read the plugin [documentation](http://plugins.krajee.com/file-input) and [demos](http://plugins.krajee.com/file-input/demo).
- [x] This is not a general programming / coding question. (Those should be directed to the [webtips Q & A forum](http://webtips.krajee.com/questions)).
- [x] I have attempted to find the simplest possible steps to reproduce the issue.
- [ ] I have included a failing test as a pull request (Optional).

## Steps to reproduce the issue

1. I am try to upload big file (8.24) GB on yii2
2. For that i am use chunk upload and next code
```php
$form->field($model, 'files[]')->widget(FileInput::classname(), [
    'options' => ['multiple' => true],
    'pluginOptions' => [
        'enableResumableUpload' => true,
        'showCancel' => true,
        'initialPreviewAsData' => true,
        'uploadAsync' => true,
        'resumableUploadOptions' => [
            'chunkSize' => 50 * 1024,
            'maxThreads' => 2
        ],
        'overwriteInitial' => false,
        'uploadUrl' => Url::to(['/files/upload']),
        'maxFileCount' => 5,
        'maxFileSize' => 10 * 1048576,
        'msgSizeTooLarge' => Yii::t('app', 'File "{name}" (<b>{customSize} KB</b>) exceeds maximum allowed upload size of <b>{customMaxSize}</b>.'),
    ]
```
3. While the file is downloading, I see not true statistics (download time and speed). Speed jumps from 20 kb/s to 2 Mb and time shwo near 24 h. But in reality, the file is loaded in 20-30 minutes.  Experiments with parameters failed
<img width="1391" alt="Снимок экрана 2020-12-03 в 10 23 34" src="https://user-images.githubusercontent.com/2761022/100983551-25a39500-3552-11eb-8366-0b7e09804f91.png">
<img width="1399" alt="Снимок экрана 2020-12-03 в 10 25 45" src="https://user-images.githubusercontent.com/2761022/100983561-289e8580-3552-11eb-9920-d9c606ed42e0.png">
<img width="1397" alt="Снимок экрана 2020-12-03 в 11 03 18" src="https://user-images.githubusercontent.com/2761022/100987562-5934ee00-3557-11eb-9613-de6132ed7439.png">

## Environment

Browsers

- [x] Google Chrome
- [ ] Mozilla Firefox
- [ ] Internet Explorer
- [ ] Safari

Operating System

- [ ] Windows
- [x] Mac OS X
- [ ] Linux
- [ ] Mobile

Libraries

- jQuery version:
- bootstrap-fileinput version:
- yii2

## Isolating the problem

- [ ] This bug happens [on the plugin demos page](http://plugins.krajee.com/file-input/demo)
- [ ] The bug happens consistently across all tested browsers
- [ ] This bug happens when using bootstrap-fileinput without other plugins
- [ ] I can reproduce this bug in [a jsbin](https://jsbin.com/)

**Comment by kartik-v** [comment]
Enhanced via above commit to show the correct bitrate calculation. Please check and let know.

**Comment by vov4ik08** [comment]
Hello. Thank you. Better but not good. If you upload one file in one stream, in parts of 50 mb, then it counts the time almost correctly. But if you upload a file in 4 streams of 20 MB, then the time counts better, but still far from the truth. Starts not counting correctly after uploading the first four parts and starting to uploding the next.
<img width="1674" alt="Снимок экрана 2020-12-09 в 14 15 17" src="https://user-images.githubusercontent.com/2761022/101628891-2e500b80-3a29-11eb-81e3-c0d64765e977.png">

**Comment by kartik-v** [comment]
Refer #1664 for a better enhancement to address this.

**Comment by kartik-v** [comment]
Resolved via #1664
